### ISSUE TITLE
Accessibility - Hamburger menu is missing a label

### ISSUE THREAD

**Issue by maur1** [DependencyTrack/frontend#1201]
### Current Behavior

Hamburger menu is missing a label describing it’s purpose to screen readers

### Proposed Behavior

Add label to the hamburger menu describing its purpose e.g "Toggle navigation menu"

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [x] I have checked the [existing issues](https://github.com/DependencyTrack/frontend/issues) for whether this enhancement was already requested

**Comment by cuteshaun** [comment]
I'd be happy to take this on. Let me know if it's available!
